controller/api/destination/watcher/endpoints_watcher.go

```diff
@@ -565,6 +806,21 @@ func (pp *portPublisher) updatePort(targetPort namedPort) {
 	}
 }
 
+func (pp *portPublisher) deleteEndpointSlice(es *dv1beta1.EndpointSlice) {
+	addrSet := pp.endpointSliceToAddresses(es)
+	for id := range pp.addresses.Addresses {
+		if _, found := addrSet.Addresses[id]; found {
+			delete(pp.addresses.Addresses, id)
```

Comment:
don't we also need to send a `Remove` update to the listener with the deleted addresses?

Reply:
this completely eluded me, added it in, thanks!